gulpfile.js

```diff
@@ -64,8 +65,8 @@ gulp.task("workers", function() {
     gulp.src(paths.workers_webpage)
         .pipe(gulp.dest("build/workers/webpage"));
 
-    gulp.src(paths.workers_pjs)
-        .pipe(gulp.dest("build/workers/pjs"));
+    // TODO(kevinb7) make this more gulp-y
```

Comment:
gulp is all about processing streams and ```bundle_deps.bundle_deps(bundle_deps.deps);``` (on the following line) doesn't really do that.  All of the dependencies are listed directly in bundle_deps.js.  It would be nice if those paths could be passed in via a glob and then have it output a stream so that additional processing could be done like minification.